packages/wallets/src/features/cfd/modals/ClientVerificationModal/ClientVerificationModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { DerivLightUploadPoiIcon } from '@deriv/quill-icons';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { Text, useDevice } from '@deriv-com/ui';
+import { ModalStepWrapper } from '../../../../components';
+import { getMarketTypeDetails, MARKET_TYPE } from '../../constants';
+import { TModifiedMT5Accounts } from '../../types';
+import { DocumentsList } from './components';
+import './ClientVerificationModal.scss';
+
+type TClientVerificationModal = {
+    account: TModifiedMT5Accounts;
+};
+
+const ClientVerificationModal: React.FC<TClientVerificationModal> = ({ account }) => {
+    const { localize } = useTranslations();
+    const { isMobile } = useDevice();
+    const { title } = getMarketTypeDetails(localize, account.product)[account.market_type || MARKET_TYPE.ALL];
+
+    return (
+        <ModalStepWrapper title={account.is_added ? localize('Verify account') : localize('Create account')}>
+            <div className='wallets-client-verification-modal'>
+                <DerivLightUploadPoiIcon height={128} width={128} />
+                <Text align='center' size={isMobile ? 'md' : 'sm'}>
```

Comment:
Is this the correct sizing for tablet view or should it be `{isDesktop ? 'sm' : 'md'}` instead?

Reply:
Yes @lubega-deriv, as per the design, the font-size is 14px for both mobile and desktop.
<img width="808" alt="Screenshot 2024-10-02 at 9 00 01 AM" src="https://github.com/user-attachments/assets/12c97aee-2df7-42d4-9e1f-1836a4a220be">
<img width="657" alt="Screenshot 2024-10-02 at 9 00 09 AM" src="https://github.com/user-attachments/assets/57a58842-6df1-408e-8848-e4c0ccca1428">